Shimakaze-chan/Commands/Info/InfoCommands.cs

```diff
@@ -11,10 +16,188 @@ class InfoCommands : Commands
+                    }
+                    // Header
+                    var userInfo = new DiscordEmbedBuilder()
+                        .WithAuthor($"{member.Username}#{member.Discriminator} ({member.Id})",
+                            "", member.AvatarUrl)
+                        .WithTimestamp(DateTime.Now)
+                        .WithColor(new DiscordColor("#3498db"))
+                        .WithThumbnail(member.AvatarUrl)
+                        .WithUrl(member.AvatarUrl)
+                        .AddField($"Status", $"```\n{member.Presence.Status}```", true);
+                    // Activities
+                    string streams = null;
+                    string games = null;
+                    foreach (var act in member.Presence.Activities)
+                    {
+                        if (act.ActivityType is ActivityType.Custom)
+                        {
+                            continue;
+                        }
+                        // So basically, am very tiny. In case you really need to know i just need a matching type lol.
+                        // It's discarded anyway and a .ToString() still executes.
+                        // Should a C style switch not have this problem it could be more elegant, if verbose af.
+                        _ = act.ActivityType switch
+                        {
+                            ActivityType.Watching => userInfo
+                                .AddField($"Watching", $"```{act.Name}```").ToString(),
+                            ActivityType.ListeningTo => userInfo
+                                .AddField($"Listening to", $"```{act.Name}```").ToString(),
+                            ActivityType.Playing => games += $"```{act.Name}```",
+                            ActivityType.Streaming => streams += $"```{act.RichPresence.Details}```\n{act.StreamUrl}",
+                            _ => throw new NotImplementedException()
+                        };
+                    }
+                    if (games?.Length > 0)
+                    {
+                        userInfo.AddField($"Playing", $"{games} ");
+                    }
+
+                    if (streams?.Length > 0)
+                    {
+                        userInfo.AddField($"Streaming", $"{streams} ");
+                    }
+                    // Account info
+                    userInfo
+                        .AddField($"Custom status",$"\n{customStatus ?? "No custom status set"}")
+                        .AddField($"Account Creation",
+                            $"```\n{member.CreationTimestamp.UtcDateTime} UTC```", false)
+                        .AddField($"Joined on", $"```\n{member.JoinedAt.UtcDateTime} UTC```")
+                        .AddField($"Server access level",
+                            $@"```{
+                                userLevel switch
+                                {
+                                    1 => "Default (1)",
```

Comment:
use the consts from ShimaConsts
```suggestion
                                    ShimaConsts.UserPermissionLevel.DEFAULT  => $"Default ({userLevel})",
```
applies to all (left side only on bot owner)